hle-survey/transformation/src/main/scala/org/broadinstitute/monster/dap/healthcondition/HealthCondition.scala

```diff
@@ -8,71 +8,68 @@ import enumeratum.values.{LongEnum, LongEnumEntry}
   * @param value raw value to store on a per-row basis in BQ
   * @param label string label to associate with the raw value in lookup tables
   * @param conditionType general category of the condition
-  * @param abbreviation short-hand used in RedCap to represent the condition
-  * @param hasCg true if the RedCap data might contain 'cg' data for the condition
-  * @param hasDx true if the RedCap data might contain 'dx' data for the condition
+  * @param both TODO
+  * @param cg TODO
+  * @param dx TODO
   * @param isOther true if the condition has a '_spec' field for free-form user entry
-  * @param cgPrefixOverride if set, will override the auto-computed value used for
-  *                         the prefix of congenital data fields
   * @param computeGate mapping function from `data prefix` => the name of the Y/N field
   *                    marking if any data exists for the condition
   */
 sealed abstract class HealthCondition(
   override val value: Long,
   val label: String,
   val conditionType: HealthConditionType,
-  val abbreviation: String,
-  val hasCg: Boolean,
-  val hasDx: Boolean,
+  val both: Option[String] = None,
+  val cg: Option[String] = None,
+  val dx: Option[String] = None,
   val isOther: Boolean = false,
-  val cgPrefixOverride: Option[String] = None,
   val computeGate: String => String = identity
```

Comment:
It looks like the references to this `computeGate` value have been removed. Do we still need it?

Reply:
I think I removed it in a later commit

Reply:
Ope, nevermind. I hadn't refreshed!